addon/metrics-adapters/azure-app-insights.js

```diff
@@ -0,0 +1,44 @@
+import { assert } from "@ember/debug";
+import BaseAdapter from "./base";
+import removeFromDOM from '../utils/remove-from-dom';
+
+export default BaseAdapter.extend({
```

Comment:
Would you mind converting this into a native class? The Facebook Pixel adapter is a good example.

Reply:
Sure no prob. I just realized that I was working off the blueprint output so I went ahead and updated the blueprint to produce native classes as well.